app/src/main/java/space/taran/arknavigator/ui/App.kt

```diff
@@ -23,4 +28,16 @@ class App: Application() {
             .appModule(AppModule(this))
             .build()
     }
+
+    override fun attachBaseContext(base: Context?) {
+        super.attachBaseContext(base)
+
+        initAcra {
+            buildConfigClass = BuildConfig::class.java
+            reportFormat = StringFormat.JSON
+            toast {
+                text = "Sorry, the app crashed. Developers will be notified."
```

Comment:
This initialization takes around 2 seconds when a crash happens.
Let's initialize it only if we have crash reporting enabled.